### ISSUE TITLE
[Windows Terminal – Settings – Startup]: 'Search for settings' text is slightly truncated after applying resize settings.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#20597]
### Windows Terminal version

1.25.1912.0

### Windows build number

28120.2738

### Other Software

**Test Environment:**
OS Version: Windows 11 version 26H1 (OS Build 28120.2738)
App Name: Windows Terminal Preview

### Steps to reproduce

**Pre-requisite:**
Apply the resize using following path:
Settings>Accessibility>Display>Display resolution>Change display resolution to '1680*1050'>Keep changes>Drag the 'Text size' slider to '200%'>Apply>Re-open the application.

**Repro Steps:**
1.	Launch the 'Windows Terminal' application.
2.	Press 'Ctrl + ,' keys.
3.	The 'Settings' page will open.
4.	Observe the 'Search for settings' text.

**User Experience:**
Users who increase the text size may have difficulty reading the complete 'Search for settings' text because part of the text is clipped.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html

**Attachment:**
https://onedrive.cloud.microsoft/:f:/a@rc32u9qu/S/IgB0LMBk67cOSblvU6oYhLlwAbgF0xUU2qu6WOXrOi4oDy4?e=mGCZiG

### Expected Behavior

The 'Search for settings' text should be fully visible and readable after applying the resize settings, without any clipping or truncation.

### Actual Behavior

After applying the resize settings, the 'Search for settings' text is slightly truncated at the bottom and is not fully visible.
